weave

```diff
@@ -48,9 +48,10 @@ usage() {
     echo "weave reset"
     echo "weave rmpeer       <peer_id>"
     echo
-    echo "where <peer>    is of the form <ip_address_or_fqdn>[:<port>], and"
-    echo "      <cidr>    is of the form <ip_address>/<routing_prefix_length>"
-    echo "      <peer_id> is a <nickname> or weave internal peer ID"
+    echo "where <peer>           is of the form <ip_address_or_fqdn>[:<port>], and"
+    echo "      <cidr>           is of the form <ip_address>/<routing_prefix_length>"
+    echo "      <peer_id>        is a <nickname> or weave internal peer ID"
+    echo "      <listen_address> is of the form [<ip_address>]:<port>"
```

Comment:
I thought we also allowed just `<ip_address>`.
And perhaps call the whole thing just `<address>`.